types/typo-js/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for typo-js 1.2
+// Project: https://github.com/cfinke/Typo.js#readme
+// Definitions by: Will Hawker <https://github.com/whawker>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface TypoSettings {
+    dictionaryPath?: string;
+    asyncLoad?: boolean;
+    loadedCallback?: (dict: Typo) => void;
+}
+
+declare class Typo {
+    constructor(dictionary?: string, affData?: string | null, wordsData?: string | null, settings?: TypoSettings);
+    loaded: boolean;
+    check: (word: string) => boolean;
+    suggest: (word: string, limit?: number) => string[];
+}
+
+export = Typo;
```

Comment:
Hoping I've got this right, as [`typo-js` uses `module.exports = Typo`](https://github.com/cfinke/Typo.js/blob/master/typo/typo.js#L1013) (and also exports a global variable)

Reply:
You can add `export as namespace Typo` to cover the global thing